scripts/incentive-spreadsheet-to-json.ts

```diff
@@ -110,11 +115,55 @@ export function spreadsheetToJson(
   });
   return {
     invalidCollectedIncentives,
+    validCollectedIncentives,
     invalidStateIncentives,
     validStateIncentives,
   };
 }
 
+type UrlParts = {
+  spreadsheetId: string;
+  incentiveDataSheetId: number;
+};
+
+const urlRegex = new RegExp(
+  /https:\/\/docs\.google\.com\/spreadsheets\/d\/(.*)\/pub\?gid=([0-9]+)&.*/,
+);
```

Comment:
Well, one reason to use the RegExp constructor is to pass it a normal string as the pattern; that way you don't have to escape all the forward slashes. You could do that here since there aren't any characters in the regex that would have special meaning in a normal quoted string.

Reply:
Done. I had gotten tripped up with the string version because I forgot you had to `\\` the question mark that starts the query params.